lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -388,27 +421,38 @@ class MongoClientInterface {
      * We also test the existence of the versionId property
      * to manage the case of an object created before the
      * versioning was enabled.
+     * @param {Object} c bucket collection
+     * @param {String} bucketName bucket name
+     * @param {String} objName object name
+     * @param {Object} objVal object metadata
```

Comment:
whenever possible, please specify the expected "type" of the Object, like:

```suggestion
     * @param {ObjectMD} objVal object metadata
```

Reply:
The `ObjectMD` example here might bring confusion cause what we get is an object that contains the ObjectMD data not the actual instance of the `ObjectMD` class